ts/Core/Chart/Chart.ts

```diff
@@ -3822,6 +3823,10 @@ class Chart {
                         // operation has finished.
                         axis.isPanning = trigger !== 'zoom';
 
+                        if (axis.isPanning && !isAnyAxisPanning) {
```

Comment:
```suggestion
                        if (axis.isPanning) {
```
It's sufficient to check for `axis.isPanning` here, right? Doesn't matter if it's already true, it doesn't get any truer.

Reply:
Yeah, definitely - my initial thought was "there is no need to assign `true` a few times". But making the file a bit smaller is also a valid reason for that.